patches/minecraft/net/minecraft/client/renderer/tileentity/TileEntityItemStackRenderer.java.patch

```diff
@@ -1,6 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/tileentity/TileEntityItemStackRenderer.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/tileentity/TileEntityItemStackRenderer.java
-@@ -118,6 +118,7 @@
+@@ -39,6 +39,7 @@
+     private final TileEntityBed field_193843_g = new TileEntityBed();
+     private final TileEntitySkull field_179023_f = new TileEntitySkull();
+     private final ModelShield field_187318_g = new ModelShield();
++    public static final java.util.Map<net.minecraftforge.registries.IRegistryDelegate<Item>, TileEntityItemStackRenderer> ITEM_TO_RENDERER = new java.util.HashMap<>();
```

Comment:
Since the keys are unique, I think this can be an `IdentityHashMap` to squeeze out a bit more performance.

Reply:
You should encapsulate this field.

Reply:
No point to make the patch bigger.  The public field is fine.